packages/acs-ui-common/package.json

```diff
@@ -38,31 +38,31 @@
     "@types/react": ">=16.8.0 <19.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "~7.16.0",
-    "@babel/core": "~7.16.0",
-    "@microsoft/api-documenter": "~7.12.11",
-    "@microsoft/api-extractor": "~7.18.0",
-    "@types/jest": "~29.5.2",
+    "@babel/cli": "^7.16.0",
+    "@babel/core": "^7.16.0",
+    "@microsoft/api-documenter": "^7.12.11",
```

Comment:
We have seen in the past that minor versions of these two components can have different behaviour. 
Perhaps leave these dev tools as ~ rather than ^?

Reply:
Agree! I can't find a very strong reason that some app will use this package on their side

Reply:
All updated!